### DISCUSSION TITLE
Minimum allowed value of 'OIDCCacheShmMax' is 1073741824 with 2.4.16.2

### DISCUSSION THREAD

**Discussion by bbartke** [OpenIDC/mod_auth_openidc#1260] [Q&A] [answered]
Hi,
after upgrading to 2.4.16.2 OIDCCacheShmMax can not longer be set to a value < 1073741824:

```
OIDCCacheType shm
OIDCCacheShmMax 2048
OIDCCacheShmEntrySizeMax 49696
```

```
AH00526: Syntax error on line ... of ...:
Invalid value for directive 'OIDCCacheShmMax': integer value 2048 is smaller than the minimum allowed value 1073741824
```

Documentation says that default value is 10000. Could this be a bug?

https://github.com/OpenIDC/mod_auth_openidc/blob/master/auth_openidc.conf#L660-L661
```
# When not specified, a default of 10000 entries is used.
#OIDCCacheShmMax <number>
```

**Comment by zandbelt** [comment, marked as answer]
ouch, that's another typo due to the configuration rewrite of 2.4.16.x; that is now fixed in 299cd158916f9f5155d8fc255195c8559d215edc and will be part of 2.4.16.3 soon; thanks for reporting

**Reply by bbartke** [reply]
Thanks for providing mod_auth_openidc to us! :metal:
